presentation/src/main/java/com/pocs/presentation/view/user/detail/UserPostListActivity.kt

```diff
@@ -0,0 +1,42 @@
+package com.pocs.presentation.view.user.detail
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.pocs.presentation.databinding.ActivityUserPostListBinding
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class UserPostListActivity : AppCompatActivity() {
+
+    private var _binding: ActivityUserPostListBinding? = null
+    private val binding: ActivityUserPostListBinding get() = requireNotNull(_binding)
+
+    companion object {
+        fun getIntent(context: Context): Intent {
+            return Intent(context, UserPostListActivity::class.java)
+        }
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        _binding = ActivityUserPostListBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+        initToolBar()
+
```

Comment:
밑에 빈 여백 지워주시고 아래처럼 한 칸 띄워주세요 
```suggestion
         setContentView(binding.root)
         
         initToolBar()
```